examples/exceptions/api_exception.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from staxapp.config import Config
+from staxapp.openapi import StaxClient
+from staxapp.exceptions import ApiException
+
+Config.access_key = os.getenv("STAX_ACCESS_KEY")
+Config.secret_key = os.getenv("STAX_SECRET_KEY")
+
+# Read all workload catalogue items within your Stax Organisation
+workloads = StaxClient("workloads")
+try:
+	response = workloads.ReadCatalogueItems()
+except ApiException as e:
+	print(e)
+	if e.status_code == 404:
+		print("No accounts exist")
```

Comment:
Is "No accounts exist" the right text here?
Seems strange to get that back from a `ReadCatalogueItems()` call?

Reply:
I've done this change